tests/kernel_performance/run_perf_case.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary: run performance cases
+# Maintainer: Joyce Na <jna@suse.de>
+
+package run_perf_case;
+use strict;
+use power_action_utils 'power_action';
+use warnings;
+use testapi;
+use base "y2logsstep";
+
+sub run_one_by_one {
+    my $case     = get_var("CASE_NAME");
+    my $time_out = get_var("RUN_TIME_MINUTES");
+
+    # create run list
+    assert_script_run("echo \'#!/bin/bash\' > /root/qaset/list");
+    assert_script_run("echo \'SQ_TEST_RUN_LIST=(\' >> /root/qaset/list");
+
+    # if customized monitor defined,add it to runlist,otherwise use default one
+    if (get_var("MONITOR")) {
+        my $monitor = get_var("MONITOR");
+        assert_script_run("echo \"$monitor\" >>/root/qaset/list");
+    }
+    assert_script_run("echo \"$case\" >> /root/qaset/list");
+    assert_script_run("echo \')\' >> /root/qaset/list");
```

Comment:
For all of the above how about to use a here-doc in the SUT shell (and a safer shebang):

```perl
my $monitor = get_var('MONITOR', '');
my $script = <<'
assert_script_run('cat - >/root/qaset/list <<"EOF"
#!/bin/sh -e
SQ_TEST_RUN_LIST=(
$monitor
$case
)
EOF
```

but honestly I doubt that the shebang is needed at all as we also do not mark the "script" as executable

Reply:
good suggestion,but maybe next version